src/status_im/ui/screens/events.cljs

```diff
@@ -109,21 +109,19 @@
 
 (re-frame/reg-fx
   :http-post
-  (fn [{:keys [action data success-event-creator failure-event-creator]}]
-    (utils/http-post action
-                     data
-                     #(re-frame/dispatch (success-event-creator %))
-                     #(re-frame/dispatch (failure-event-creator %)))))
-
-(defn- http-get [{:keys [url response-validator success-event-creator failure-event-creator]}]
-  (if response-validator
-    (utils/http-get url
-                    response-validator
-                    #(re-frame/dispatch (success-event-creator %))
-                    #(re-frame/dispatch (failure-event-creator %)))
-    (utils/http-get url
-                    #(re-frame/dispatch (success-event-creator %))
-                    #(re-frame/dispatch (failure-event-creator %)))))
+  (fn [{:keys [action data success-event-creator failure-event-creator timeout-ms]}]
+    (let [on-success #(re-frame/dispatch (success-event-creator %))
+          on-error   #(re-frame/dispatch (failure-event-creator %))]
```

Comment:
Could we do something similar to `http-get` and have last argument a map? This way we can also use a static map (and no cond).
Sorry missed that one!

Reply:
@jeluard fixed now.